lesson_03/homework_03.py

```diff
@@ -16,7 +22,16 @@
 моря становить 37 800 км2. Яку площу займають Чорне та Азов-
 ське моря разом?
 """
+print("\ntask 04")
+square_black_sea = int(436402)
+square_azov_sea = int(37800)
```

Comment:
```suggestion
square_black_sea = 436402
square_azov_sea = 37800
```
тут і так інт, нічого додатково робити не треба

Reply:
Змінила